cmd/complytime/cli/generate.go

```
@@ -29,15 +30,21 @@ func generateCmd(common *option.Common) *cobra.Command {
 		Short:   "Generate PVP policy from an assessment plan",
 		Example: "complytime generate",
 		Args:    cobra.NoArgs,
+		PreRun: func(cmd *cobra.Command, _ []string) {
+			enableDebug(logger, common)
+		},
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			return runGenerate(cmd, generateOpts)
+			if err := runGenerate(cmd, generateOpts, logger); err != nil {
+				logger.Error(err.Error())
```

Not returning the error here is causing the application the exit code to be zero even there is an error. There may be a way to silence the error so you can still log it and it will exit with one.